packages/server/src/ui/routes/build-view/lhr-comparison.jsx

```diff
@@ -113,13 +113,14 @@ export function computeAuditGroups(lhr, baseLhr, options) {
   return auditGroups;
 }
 
-/** @param {{hookElements: LHCI.HookElements<'dropdowns'>, selectedAuditId?: string | null, lhr?: LH.Result, baseLhr?: LH.Result, percentAbsoluteDeltaThreshold: number, setPercentAbsoluteDeltaThreshold: (x: number) => void}} props */
```

Comment:
I think we can leave this just as `dropdowns` it was already set to be plural for more than one. 

use a fragment when it's used instead?

Reply:
Do you mean like this?
```
<LhrComparison
  ...
  hookElements={{
    ...
    dropdowns: (
      <Fragment>
        <Dropdown
          label="Base Url"
          className="dropdown--url dropdown--base-url"
          value={baseUrl}
          setValue={url => {
            const to = new URL(window.location.href);
            to.searchParams.set('baseUrl', url);
            to.searchParams.set('compareUrl', compareUrl);
            route(`${to.pathname}${to.search}`);
          }}
          options={availableUrlOptions}
        />
        <Dropdown
          label="Compare Url"
          className="dropdown--url dropdown--compare-url"
          value={compareUrl}
          setValue={url => {
            const to = new URL(window.location.href);
            to.searchParams.set('baseUrl', baseUrl);
            to.searchParams.set('compareUrl', url);
            route(`${to.pathname}${to.search}`);
          }}
          options={availableUrlOptions}
        />
      </Fragment>
    ),
  }}
/>
```